Currently the live version does not build because of this dependency.  Remove it for now

--- a/vncviewer/CMakeLists.txt
+++ b/vncviewer/CMakeLists.txt
@@ -81,7 +81,6 @@
                 --desktop --template vncviewer.desktop.in
                 -d ${CMAKE_SOURCE_DIR}/po -o vncviewer.desktop
       DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/vncviewer.desktop.in
-              ${CMAKE_SOURCE_DIR}/po/*.po
     )
   elseif(INTLTOOL_MERGE_EXECUTABLE)
     add_custom_command(OUTPUT vncviewer.desktop
@@ -93,7 +92,6 @@
                 -d ${CMAKE_SOURCE_DIR}/po
                 vncviewer.desktop.intl vncviewer.desktop
       DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/vncviewer.desktop.in
-              ${CMAKE_SOURCE_DIR}/po/*.po
     )
   else()
     add_custom_command(OUTPUT vncviewer.desktop
@@ -110,7 +108,6 @@
                 --xml --template ${CMAKE_CURRENT_SOURCE_DIR}/org.tigervnc.vncviewer.metainfo.xml.in
                 -d ${CMAKE_SOURCE_DIR}/po -o org.tigervnc.vncviewer.metainfo.xml
       DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/org.tigervnc.vncviewer.metainfo.xml.in
-              ${CMAKE_SOURCE_DIR}/po/*.po
     )
   elseif(INTLTOOL_MERGE_EXECUTABLE)
     add_custom_command(OUTPUT org.tigervnc.vncviewer.metainfo.xml
@@ -123,7 +120,6 @@
                 -x ${CMAKE_SOURCE_DIR}/po
                 org.tigervnc.vncviewer.metainfo.xml.intl org.tigervnc.vncviewer.metainfo.xml
       DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/org.tigervnc.vncviewer.metainfo.xml.in
-              ${CMAKE_SOURCE_DIR}/po/*.po
     )
   else()
     add_custom_command(OUTPUT org.tigervnc.vncviewer.metainfo.xml
